vms/platformvm/block/executor/acceptor_test.go

```diff
@@ -270,72 +276,55 @@ func TestAcceptorVisitCommitBlock(t *testing.T) {
 		bootstrapped: &utils.Atomic[bool]{},
 	}
 
-	blk, err := block.NewApricotCommitBlock(parentID, 1 /*height*/)
+	blk, err := block.NewBanffCommitBlock(blkTime, parentID, parentHeight+1)
 	require.NoError(err)
-
-	err = acceptor.ApricotCommitBlock(blk)
-	require.ErrorIs(err, state.ErrMissingParentState)
-
-	// Set [blk]'s parent in the state map.
-	parentOnAcceptState := state.NewMockDiff(ctrl)
-	parentOnAbortState := state.NewMockDiff(ctrl)
-	parentOnCommitState := state.NewMockDiff(ctrl)
-	parentStatelessBlk := block.NewMockBlock(ctrl)
-	parentState := &blockState{
-		statelessBlock: parentStatelessBlk,
-		onAcceptState:  parentOnAcceptState,
-		proposalBlockState: proposalBlockState{
-			onAbortState:  parentOnAbortState,
-			onCommitState: parentOnCommitState,
-		},
-	}
-	acceptor.backend.blkIDToState[parentID] = parentState
-
 	blkID := blk.ID()
-	// Set expected calls on dependencies.
-	// Make sure the parent is accepted first.
-	gomock.InOrder(
-		parentStatelessBlk.EXPECT().ID().Return(parentID).Times(2),
-		s.EXPECT().SetLastAccepted(parentID).Times(1),
-		parentStatelessBlk.EXPECT().Height().Return(blk.Height()-1).Times(1),
-		s.EXPECT().SetHeight(blk.Height()-1).Times(1),
-		s.EXPECT().AddStatelessBlock(parentState.statelessBlock).Times(1),
-
-		s.EXPECT().SetLastAccepted(blkID).Times(1),
-		s.EXPECT().SetHeight(blk.Height()).Times(1),
-		s.EXPECT().AddStatelessBlock(blk).Times(1),
-	)
 
-	err = acceptor.ApricotCommitBlock(blk)
-	require.ErrorIs(err, errMissingBlockState)
-
-	// Set [blk]'s state in the map as though it had been verified.
-	acceptor.backend.blkIDToState[parentID] = parentState
-	onAcceptState := state.NewMockDiff(ctrl)
-	acceptor.backend.blkIDToState[blkID] = &blockState{
-		onAcceptState: onAcceptState,
+	{
+		err = acceptor.BanffCommitBlock(blk)
```

Comment:
updated to BanffCommitBlock here. There is not much difference in the logic and it's better to test options currently created on mainnet rather than deprecated versions